server/request_processor.go

```diff
@@ -53,6 +58,9 @@ func (r *RpcRequest) ProcessRequest() *types.JsonRpcResponse {
 
 	switch {
 	case r.jsonReq.Method == "eth_sendRawTransaction":
+		r.reqRecord.ethSendRawTxEntry.IsWhiteHatBundleCollection = r.isWhitehatBundleCollection
```

Comment:
`r.reqRecord.NewEthSendRawTxEntry`, and keep the instance around (probably as part of the request)